src/features/aptosSignTransaction.ts

```diff
@@ -1,30 +1,82 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-import { AnyRawTransaction, AccountAuthenticator } from '@aptos-labs/ts-sdk'
+import {
+  AccountAddress,
+  AccountAuthenticator,
+  AnyRawTransaction,
+  InputGenerateTransactionPayloadData,
+  Network,
+  PublicKey,
+  TransactionPayload
+} from '@aptos-labs/ts-sdk'
 import { UserResponse } from '../misc'
 
-/** Version of the feature. */
-export type AptosSignTransactionVersion = '1.0.0'
+// region Feature definition
+
 /** Name of the feature. */
 export const AptosSignTransactionNamespace = 'aptos:signTransaction'
-/**
- * A Wallet Standard feature for signing a Aptos transaction, and returning the
- * account authenticator.
- */
-export type AptosSignTransactionFeature = {
-  /** Namespace for the feature. */
+
+export type AptosSignTransactionFeatureV1_0 = {
   [AptosSignTransactionNamespace]: {
-    /** Version of the feature API. */
-    version: AptosSignTransactionVersion
+    version: '1.0.0',
     signTransaction: AptosSignTransactionMethod
   }
 }
 
+export type AptosSignTransactionFeatureV1_1 = {
+  [AptosSignTransactionNamespace]: {
+    version: '1.1'
+    signTransaction: AptosSignTransactionMethod & AptosSignTransactionMethodV1_1
```

Comment:
Since this is a minor bump, we should support exactly the same input/output as before, and can add to it.
It works exactly as expected when implementing the interface e.g. when using 1.1 [you NEED to specify both interfaces](https://github.com/aptos-labs/aptos-connect/blob/b0fff1c3e11e39b0f0071a56279ceb1907dba1ca/packages/wallet-adapter-plugin/src/AptosConnectWallet.ts#L268-L270)

and eventually, version 2.0 will drop old interfaces